src/main/java/eu/dissco/core/translator/service/DwcaService.java

```diff
@@ -131,8 +131,9 @@ public TranslatorJobResult retrieveData() {
   }
 
   public void getSpecimenData(Set<String> ids, Archive archive, AtomicInteger processedRecords)
-      throws JsonProcessingException {
```

Comment:
it doesn't look like storeEmlRecord throws an IO exception. Why make this more generic than it needs to be?

Reply:
The Files.readBytes can throw a IOException so the storeEmlRecord can throw it, so I decided to make the exception more generic.